hrjobcontract/css/hrjc.css

```diff
@@ -11572,29 +11572,10 @@ fieldset[disabled]
 }
 /* line 1, ../scss/hrjc/partials/_loader.scss */
 #hrjob-contract .hrjc-loader {
-  background: rgba(255, 255, 255, 0.5) url("../img/loading-overlay.gif") no-repeat center center;
-  -moz-border-radius: 0;
-  -webkit-border-radius: 0;
-  border-radius: 0;
   position: absolute;
-  width: 100%;
-  height: 100%;
-  top: 0;
-  left: 0;
+  top: 50%;
```

Comment:
Not quite sure this centers the loader perfectly. Could you check please?

Reply:
Well, tours out `top: 50%; left:50%` is not enough for positioned element to keep in center. So, I have created a fiddle http://jsfiddle.net/6xo11zwv/266/ with `transform: translate(-50%, -50%);` as well. I have checked with different browsers and it works. I have update the the style.